hbase-server/src/main/java/org/apache/hadoop/hbase/util/TableDescriptorChecker.java

```diff
@@ -186,9 +178,18 @@ public static void sanityCheck(final Configuration c, final TableDescriptor td)
       }
 
       // check replication scope
-      checkReplicationScope(hcd);
+      try {
```

Comment:
it would be nice to keep these check methods somewhat consistent. You removed the warnOrThrowExceptionForFailure from the above checks, but added them here. Can you instead keep it consistent (all here or all in the check methods)?

That might mean either removing this diff here, or adding back the try/catches above.